libraries/net/include/graphene/net/node.hpp

```diff
@@ -292,8 +319,11 @@ namespace graphene { namespace net {
 
         void disable_peer_advertising();
         fc::variant_object get_call_statistics() const;
-      private:
+      protected:
         std::unique_ptr<detail::node_impl, detail::node_impl_deleter> my;
+      protected:
```

Comment:
Can we remove this line?

Reply:
This is used to make testing easier. So unless we want to redo the test, I would recommend we keep it.

Reply:
Sorry, my eyes failed me and I didn't see it was in there twice. Fixed.